### ISSUE TITLE
Migration does nothing and produces no error

### ISSUE THREAD

**Issue by AhmedIbrahim88** [anthonylv/pyD2W#5]
![Screenshot 2019-06-11 14 02 28](https://user-images.githubusercontent.com/51604990/59270436-bcd7a700-8c51-11e9-9c6e-89d50910973b.png)

![image](https://user-images.githubusercontent.com/51604990/59270473-dc6ecf80-8c51-11e9-8332-a25cd51b915f.png)

it Migrates from but when checking the wordpress database its untouched nothing happens I hope that you can help please

**Comment by anthonylv** [comment]
Hi Ahmed,

This script is designed to help migrate highly-customised Drupal installations but it's necessary to supply it with the SQL queries to match your setup. Looking at the screenshot of the script output and your settings file, it seems that you haven't made any adjustments.

Each Drupal installation can have different sets of tables depending on where the content is located, which modules are in use and the kinds of custom content types that have been set up. Further, each content type may have custom fields located in various tables. The names of these tables are specific to the Drupal installation. You have to tell the script where everything is located using a set of supporting SQL queries.

An example of the SQL queries you need to perform can be found in [sql-sample/migration_standard.sql](https://github.com/anthonylv/pyD2W/blob/master/sql-sample/migration_standard.sql). You will probably also need to read through the following resources to help get you started:

- [Drupal to WordPress migration SQL queries explained](https://anothercoffee.net/drupal-to-wordpress-migration-sql-queries-explained/)
- [How to prepare for your Drupal to WordPress migration project](https://anothercoffee.net/prepare-drupal-wordpress-migration-project/)
- [Drupal to WordPress Migration Guide](
https://anothercoffee.net/drupal-to-wordpress-migration-guide/)
- [Drupal to WordPress migration checklist](https://anothercoffee.net/drupal-to-wordpress-migration-checklist/)

Hope this helps,
Anthony

**Comment by AhmedIbrahim88** [comment]
Hi Anthony,

thank you for your kind reply the this is I have never dealt with drupal
and I am trying to migrate the blog posts only with their Main Image into
wordpress tried the sample_migration found but it gives me an error saying
that acc_wp_comments table was not found although the table does exist in
the wordpress database


Many Thanks,
Ahmed

On June 11, 2019 at 2:42:55 PM, Anthony Lopez-Vito (notifications@github.com)
wrote:

Hi Ahmed,

This script is designed to help migrate highly-customised Drupal
installations but it's necessary to supply it with the SQL queries to match
your setup. Looking at the screenshot of the script output and your
settings file, it seems that you haven't made any adjustments.

Each Drupal installation can have different sets of tables depending on
where the content is located, which modules are in use and the kinds of
custom content types that have been set up. Further, each content type may
have custom fields located in various tables. The names of these tables are
specific to the Drupal installation. You have to tell the script where
everything is located using a set of supporting SQL queries.

An example of the SQL queries you need to perform can be found in
sql-sample/migration_standard.sql
<https://github.com/anthonylv/pyD2W/blob/master/sql-sample/migration_standard.sql>.
You will probably also need to read through the following resources to help
get you started:

   - Drupal to WordPress migration SQL queries explained
   <https://anothercoffee.net/drupal-to-wordpress-migration-sql-queries-explained/>
   - How to prepare for your Drupal to WordPress migration project
   <https://anothercoffee.net/prepare-drupal-wordpress-migration-project/>
   - Drupal to WordPress Migration Guide
   <https://anothercoffee.net/drupal-to-wordpress-migration-guide/>
   -Drupal to WordPress migration checklist
   <https://anothercoffee.net/drupal-to-wordpress-migration-checklist/>

Hope this helps,
Anthony

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/anthonylv/pyD2W/issues/5?email_source=notifications&email_token=AMJW37T5NGKNW33JC34AEHLPZ6MU7A5CNFSM4HW5S442YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODXM7PJQ#issuecomment-500823974>,
or mute the thread

**Comment by anthonylv** [comment]
Hi Ahmed,

This tool is really only useful for people who have some detailed knowledge of Drupal and WordPress databases. Unfortunately it probably won't be much use to you if you've never dealt with Drupal. You also can't just run the sample migration file and expect some migration results. There is a lot of setup and information gathering needed beforehand, which I've documented in the above links.

For example, the acc_wp_comments table will not normally be found in your WordPress database. It is an intermediary working table that I place in my Drupal database. Where you place it (and if you even need it) would depend on how you want to perform your migration.

My guides will offer you a starting point but if you are new to Drupal and to database migrations, it might take you a long time to reach a stage where you have a working migration. In this case, you might be better off using one of the alternative paid-for plugins and commercial migration services.

Regards,
Anthony

**Comment by AhmedIbrahim88** [comment]
Thank you very much you have been more than helpful I was able to run the
migrations I was after properly after a few tries .. as you said its not an
out of the box solution but it does make it happen!

thanks a lot for the tool and your time




On June 11, 2019 at 3:36:19 PM, Anthony Lopez-Vito (notifications@github.com)
wrote:

Hi Ahmed,

This tool is really only useful for people who have some detailed knowledge
of Drupal and WordPress databases. Unfortunately it probably won't be much
use to you if you've never dealt with Drupal. You also can't just run the
sample migration file and expect some migration results. There is a lot of
setup and information gathering needed beforehand, which I've documented in
the above links.

For example, the acc_wp_comments table will not normally be found in your
WordPress database. It is an intermediary working table that I place in my
Drupal database. Where you place it (and if you even need it) would depend
on how you want to perform your migration.

My guides will offer you a starting point but if you are new to Drupal and
to database migrations, it might take you a long time to reach a stage
where you have a working migration. In this case, you might be better off
using one of the alternative paid-for plugins and commercial migration
services.

Regards,
Anthony

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/anthonylv/pyD2W/issues/5?email_source=notifications&email_token=AMJW37UW6BENVKGG6LTTUXDPZ6S5FA5CNFSM4HW5S442YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODXNENUQ#issuecomment-500844242>,
or mute the thread
